src/components/content/order/Services.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { FormOutlined } from '@ant-design/icons';
+import '../../../styles/service_order.css';
+import { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { createServicePageRoute } from '../../utils/constants';
+import { serviceVendorApi } from '../../../xpanse-api/xpanseRestApiClient';
+
+function Services(): JSX.Element {
+    const [services, setServices] = useState<{ name: string; content: string; icon: string }[]>([]);
+    const navigate = useNavigate();
+    const location = useLocation();
+
+    const onClicked = function (cfg: string) {
+        navigate(
+            createServicePageRoute
+                .concat('?serviceName=', location.hash.split('#')[1])
+                .concat('&name=', cfg.replace(' ', ''))
+        );
+    };
+
+    useEffect(() => {
+        const categoryName = location.hash.split('#')[1];
+        if (!categoryName) {
+            return;
+        }
+        serviceVendorApi.listRegisteredServicesTree(categoryName).then((rsp) => {
+            if (rsp.length > 0) {
+                let serviceList: { name: string; content: string; icon: string }[] = [];
+                rsp.forEach((item) => {
+                    let serviceItem = {
+                        name: item.name || '',
+                        content: item.versions[0].cloudProvider[0].details[0].description,
+                        icon: item.versions[0].cloudProvider[0].details[0].icon,
+                    };
+                    serviceList.push(serviceItem);
+                });
+                setServices(serviceList);
+            } else {
+                return;
+            }
+        });
+    }, [location]);
+
+    return (
+        <div className={'services-content'}>
+            <div className={'content-title'}>
+                <FormOutlined />
+                &nbsp;Select Service
```

Comment:
would it make sense to display "Select Service" only when there is any service available to to be ordered for the particular category? otherwise display "No services available"

Reply:
fixed.